hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import bottle
+from dataclasses import dataclass, asdict
+import os
+from mypy_boto3_dynamodb.service_resource import Table
+import typing as t
+
+from hmalib.models import PDQMatchRecord
+from .middleware import jsoninator, JSONifiable
+
+
+# TODO: Remove dependency on image folder key, use a common interface for
+# s3 and non-s3 HMA requests.
+IMAGE_FOLDER_KEY = os.environ["IMAGE_FOLDER_KEY"]
```

Comment:
I think for now at least we want to keep all uses of os.environ are in the same module as `lambda_handler`

Reply:
Agree, that's what the TODO is for. 

I plan to remove this soon. We'll need to do this for v0 when API uploads might make it necessary to **not** think about IMAGE_FOLDERs.

Reply:
It's really easy to break the whole API silently atm like had to be fixed in #482 I feel strongly about being consistent with this. Can you add this (or a better CI test on os.environ) to the set of work before merge like the UI diff?

Reply:
Sure. Let me cook something up.